Bottom: 74e1c002728807c89b5890c5b09ee036efc355f6
Top:    83708b08e24a5afb82b328e0c50f0afe4563fd38
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 20:25:41 +0100

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 429fbf8..1d5773b 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -612,7 +612,7 @@ void
 ghid_invalidate_all ()
 {
   if (ghidgui->need_restore_crosshair)
-    RestoreCrosshair (FALSE);
+    RestoreCrosshair ();
   ghidgui->need_restore_crosshair = FALSE;
   ghid_draw_area_update (gport, NULL);
 }
@@ -988,7 +988,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
                              -port->view_x0,
                 ghid_flip_y ? port->view_y0 - PCB->MaxHeight :
                              -port->view_y0, 0);
-  DrawAttached (TRUE);
+  DrawAttached ();
   DrawMark (TRUE);
   hidgl_flush_triangles (&buffer);
   glPopMatrix ();
